code/game/machinery/colony_floodlights.dm

```diff
@@ -94,19 +94,24 @@
 #define FLOODLIGHT_REPAIR_SCREW 4
 
 /obj/structure/machinery/colony_floodlight
-	name = "Colony Floodlight"
+	name = "colony floodlight"
 	icon = 'icons/obj/structures/machinery/big_floodlight.dmi'
 	icon_state = "flood_s_off"
 	density = TRUE
 	anchored = TRUE
 	layer = ABOVE_XENO_LAYER
-	var/damaged = 0 //Can be smashed by xenos
-	var/is_lit = 0 //whether the floodlight is switched to on or off. Does not necessarily mean it emits light.
+	///Whether it has been smashed by xenos
+	var/damaged = FALSE
+	///whether the floodlight is switched to on or off. Does not necessarily mean it emits light.
+	var/is_lit = FALSE
 	unslashable = TRUE
 	unacidable = TRUE
-	var/power_tick = 50 // power each floodlight takes up per process
+	///The power each floodlight takes up per process
+	var/power_tick = 50
 	use_power = USE_POWER_NONE //It's the switch that uses the actual power, not the lights
-	var/obj/structure/machinery/colony_floodlight_switch/fswitch = null //Reverse lookup for power grabbing in area
+	needs_power = FALSE
+	///Reverse lookup for power grabbing in area
+	var/obj/structure/machinery/colony_floodlight_switch/fswitch = null
 	var/lum_value = 7
```

Comment:
Would you mind also moving these var declarations to below the inherited ones? They're all a bit jumbled together at the moment.